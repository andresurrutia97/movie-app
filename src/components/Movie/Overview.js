import React from 'react';
import PropTypes from 'prop-types';

const Overview = ({ overview }) => {
  return <p className="text-sm text-justify text-white">{overview}</p>;
};

Overview.propTypes = {
  overview: PropTypes.string,
};

Overview.defaultProps = {
  overview: '',
};

export default Overview;
