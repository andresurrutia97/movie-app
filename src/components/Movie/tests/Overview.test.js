import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';

import Overview from '../Overview';

describe('<Overview/>', () => {
  const component = <Overview overview="overview" />;

  it('should render correctly ', () => {
    const tree = renderer.create(component).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should render a <p> tag with a text equal to "overview"', () => {
    const wrapper = shallow(component);
    expect(wrapper.props().children).toBe('overview');
  });
});
