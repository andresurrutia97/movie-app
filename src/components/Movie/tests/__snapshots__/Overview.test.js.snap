// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Overview/> should render correctly  1`] = `
<p
  className="text-sm text-justify text-white"
>
  overview
</p>
`;
